<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepperMotor</class>
 <widget class="QWidget" name="StepperMotor">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StepperMotor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../MainWindow/images.qrc">
    <normaloff>:/ico/res/ico/ing10.ico</normaloff>:/ico/res/ico/ing10.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>步进电机1</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>201</x>
         <y>320</y>
         <width>301</width>
         <height>141</height>
        </rect>
       </property>
       <property name="title">
        <string>位置控制模式</string>
       </property>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>细分数32</string>
         </property>
        </item>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
QSpinBox
{
	border:1px solid #7A7A7A;
}

QSpinBox::up-button,QSpinBox::down-button
{
	width:0px;
}
</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>脉冲数</string>
        </property>
       </widget>
       <widget class="QPushButton" name="posStart">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>启动</string>
        </property>
       </widget>
       <widget class="QPushButton" name="posStop">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>100</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>圈数:</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="prefix">
         <string>速度值</string>
        </property>
        <property name="suffix">
         <string>rpm</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-600.000000000000000</double>
        </property>
        <property name="maximum">
         <double>600.000000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>201</x>
         <y>150</y>
         <width>299</width>
         <height>141</height>
        </rect>
       </property>
       <property name="title">
        <string>速度控制模式</string>
       </property>
       <widget class="QDial" name="dial">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>20</y>
          <width>121</width>
          <height>121</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-60</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="spdStart">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>启动</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="spdStop">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>使用旋钮控制电机转速</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>10</y>
         <width>361</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>通用输入输出状态</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>340</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="3">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>IN1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>IN1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>IN1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>IN1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>IN1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>IN1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>IN1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>IN1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>IN1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>IN1</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>OUT1</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>IN1</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="pushButton_10">
           <property name="text">
            <string>OUT1</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>OUT1</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>OUT1</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>IN1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>201</x>
         <y>50</y>
         <width>301</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>点动控制</string>
       </property>
       <widget class="QPushButton" name="btnJogCtrl">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>点动控制</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spbJogSpd">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="prefix">
         <string>速度值</string>
        </property>
        <property name="suffix">
         <string>rpm</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-600.000000000000000</double>
        </property>
        <property name="maximum">
         <double>600.000000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>228</y>
         <width>171</width>
         <height>231</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QGroupBox" name="QGBSerialConfig">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>11</y>
         <width>171</width>
         <height>211</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>211</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>211</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox{
border-width:1px;
border-style:solid;
border-radius:4px;
font:bold ; 
}</string>
       </property>
       <property name="title">
        <string>串口设置</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QPushButton" name="btnOpenPort">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>69</x>
          <y>160</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>打开串口</string>
        </property>
        <property name="icon">
         <iconset resource="../../MainWindow/images.qrc">
          <normaloff>:/action/res/action/centrejust.png</normaloff>
          <normalon>:/action/res/action/stock_stop.png</normalon>:/action/res/action/centrejust.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>151</width>
          <height>136</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="3" column="0">
          <widget class="QLabel" name="lblVerify">
           <property name="text">
            <string>校验</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cbbDataBit">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblPortNum">
           <property name="text">
            <string>端口号</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblStopBit">
           <property name="text">
            <string>停止位</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblBaudRate">
           <property name="text">
            <string>波特率</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbbPortNum">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maxVisibleItems">
            <number>20</number>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <property name="minimumContentsLength">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cbbVerify">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="cbbStopBit">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cbbBaud">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblDataBit">
           <property name="text">
            <string>数据位</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="btnRefresh">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>160</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>刷新端口号</string>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
        <property name="icon">
         <iconset resource="../../MainWindow/images.qrc">
          <normaloff>:/action/res/action/stock_refresh.png</normaloff>:/action/res/action/stock_refresh.png</iconset>
        </property>
       </widget>
       <widget class="QCheckBox" name="chkFlowCtrlDTR">
        <property name="geometry">
         <rect>
          <x>31</x>
          <y>190</y>
          <width>43</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>DTR</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="chkFlowCtrlRTS">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>190</y>
          <width>43</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>RTS</string>
        </property>
       </widget>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_2">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>230</y>
         <width>361</width>
         <height>231</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;第三方的法师打发&lt;span style=&quot; font-weight:600;&quot;&gt;斯蒂芬是打发爱的是的发生的&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;法表现非常vcxvcxvxczxcvzxc &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;法规规范的非官方公司的法国&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;法国分隔符苟富贵&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/action/res/action/document-open-recent.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>201</x>
         <y>11</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnConnect">
          <property name="text">
           <string>连接设备</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblDeviceName">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>No connected devices</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../MainWindow/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>posStop</sender>
   <signal>clicked()</signal>
   <receiver>spdStop</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
